src/equations/numerical_fluxes.jl

```diff
@@ -221,6 +221,46 @@ See [`FluxLaxFriedrichs`](@ref).
 """
 const flux_lax_friedrichs = FluxLaxFriedrichs()
 
+"""
+    DissipationLaxFriedrichsEntropyVariables(max_abs_speed=max_abs_speed_naive)
+
+Create a local Lax-Friedrichs-type dissipation operator that is provably entropy stable. This operator
+must be used together with an entropy-conservative two-point flux function (e.g., `flux_ec`) to yield 
+an entropy-stable surface flux. The surface flux function can be initialized as:
+```
+flux_es = FluxPlusDissipation(flux_ec, DissipationLaxFriedrichsEntropyVariables())
+```
+
+In particular, the numerical flux has the form
+```math
+f^{\\mathrm{ES}} = f^{\\mathrm{EC}} - \\frac{1}{2} \\lambda_{\\mathrm{max}} H (w_r - w_l),
+````
+where ``f^{\\mathrm{EC}}`` is the entropy-conservative two-point flux function (computed with, e.g., `flux_ec`), ``\\lambda_{\\mathrm{max}}`` 
```

Comment:
```suggestion
f^{\mathrm{ES}} = f^{\mathrm{EC}} - \frac{1}{2} \lambda_{\mathrm{max}} H (w_r - w_l),
```
where ``f^{\mathrm{EC}}`` is the entropy-conservative two-point flux function (computed with, e.g., `flux_ec`), ``\lambda_{\mathrm{max}}`` 
```
In `@docs raw` mode only one backslash is needed.